src/views/blocks/rsvp/form/form.php

```diff
@@ -8,53 +8,52 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link    {INSERT_ARTICLE_LINK_HERE}
+ * @link    https://m.tri.be/1amp Help article for RSVP & Ticket template files.
  *
  * @since   4.9
  * @since   4.12.0 Add $post_id to filter for hiding opt-outs.
  * @since   4.12.3 Add comments to help IDE auto-completion. Array short syntax.
+ * @since   TBD Add docblock vars and use $ticket->ID instead of duplicative $ticket_id.
  *
- * @version 4.12.3
+ * @version TBD
+ *
+ * @var Tribe__Tickets__Editor__Template $this       Template object.
+ * @var int                              $post_id    [Global] The current Post ID to which RSVPs are attached.
+ * @var bool                             $must_login [Global] True if login is required and user is not logged in..
+ * @var Tribe__Tickets__Ticket_Object    $ticket     The ticket object with provider set to RSVP.
+ * @var string                           $going      The RSVP status at time of add/edit (e.g. 'yes'), or empty if not in that context.
  */
 
-$ticket_id = $this->get( 'ticket_id' );
-$post_id   = $this->get( 'post_id' );
-$going     = $this->get( 'going' );
-
 /** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
 $tickets_handler = tribe( 'tickets.handler' );
-$ticket_data     = $tickets_handler->get_object_connections( $ticket_id );
 
-$event_id = $ticket_data->event;
+$ticket_data = $tickets_handler->get_object_connections( $ticket->ID );
 
-/** @var Tribe__Tickets__RSVP $rsvp */
-$rsvp       = tribe( 'tickets.rsvp' );
-$must_login = ! is_user_logged_in() && $rsvp->login_required();
+$event_id   = $ticket_data->event;
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 3 spaces

<sub>hash: 003b172774e5d8437b61e2effaed6eb6</sub>